docs/source/implementations/codependence.rst

```diff
@@ -110,4 +110,23 @@ Let's see how various metrics work for different variable dependencies:
         ax.plot(x, y, 'ro')
         plt.savefig('{}.png'.format(dependency))
 
-::
\ No newline at end of file
+::
```

Comment:
This whole file is outdated. Have you recently pulled from dev and fixed the merge conflicts?